tests/test_elasticsearch.py

```diff
@@ -80,31 +76,22 @@ def load_json_file(filename, mode="r"):
     return json_content
 
 
-class TestElasticsearch(unittest.TestCase):
+class TestElasticsearch(TestBaseElasticSearch):
     """Base class to test new_functions.py"""
 
     maxDiff = None
 
     @classmethod
     def setUpClass(cls):
-
-        cls.es = Elasticsearch(hosts=ES_URL)
-
-        feed_backend(ES_URL, clean=True, fetch_archive=False, backend_name=BACKEND,
-                     backend_params=BACKEND_PARAMS, es_index=GIT_RAW_INDEX,
-                     es_index_enrich=GIT_ENRICH_INDEX, project=None, arthur=False)
-
-        enrich_backend(ES_URL, clean=True, backend_name=BACKEND,
-                       backend_params=BACKEND_PARAMS,
-                       ocean_index=GIT_RAW_INDEX,
-                       ocean_index_enrich=GIT_ENRICH_INDEX,
-                       unaffiliated_group=None)
+        cls.name = NAME
+        cls.enrich_index = ENRICH_INDEX
+        super().setUpClass()
 
     def setUp(self):
         """Set up the necessary functions to run unittests"""
 
         self.es = Elasticsearch(ES_URL)
```

Comment:
'self.es' could be removed? the TestBaseElasticSearch should already contain a similar variable.

Reply:
@valeriocos the `es` variable in TestBaseElasticSearch:
```python
@classmethod
    def setUpClass(cls):
        cls.es = Elasticsearch([ES_URL], timeout=3600, max_retries=50, retry_on_timeout=True)
```
is a part of the `classmethod` and cannot be accessed in the instance method in TestElasticsearch:
```python
def setUp(self):
        """Set up the necessary functions to run unittests"""

        self.es = Elasticsearch(ES_URL)
```
Right?

Reply:
Oh, nevermind, I can access it using `TestElasticseach.es`. Thanks for pointing this out!